recipes-thirdparty/dab-adapter/files/0001-PREMIUMAPP-3142-Change-keymap-to-better-reflect-RDK-.patch

```diff
@@ -0,0 +1,43 @@
+From cddff0b24e294496853d465d0ab3af853ed952a2 Mon Sep 17 00:00:00 2001
```

Comment:
Please include upstream PR reference in each patch itself as a comment so that once those changes are available, we can clean this up.